src/main/java/io/qdrant/client/VectorsFactory.java

```diff
@@ -18,13 +19,13 @@ private VectorsFactory() {
 
 	/**
 	 * Creates named vectors
-	 * @param values A map of vector names to values
+	 * @param values A map of vector names to {@link Vector}
 	 * @return a new instance of {@link Vectors}
 	 */
-	public static Vectors namedVectors(Map<String, List<Float>> values) {
+	public static Vectors namedVectors(Map<String, Vector> values) {
```

Comment:
What form of versioning is the team following for clients? This is an API breaking change in a non-prerelease versioned package, so should be handled according to the versioning scheme chosen.

At a minimum, I'd suggest calling this out as a breaking change in the next release notes.

Reply:
The major and minor versions of clients are to be in parity with the Qdrant version, i.e. `1.7`. The patch versions can be bumped.

I will include the notice in the Release Notes.